# test information
# time 2016/08/16
# add by huanglilongwk@outlook.com 
# for testing custom mavlink message

std_msgs/Header header

float32 test